### PR TITLE
refactor(yaml): align additional error messages

### PR DESCRIPTION
**Pull request by irbull**
Aligns the error messages in the `yaml` folder to match the style guide.

https://github.com/denoland/std/issues/5574

### CODE REVIEW

File: yaml/_dumper_state.ts [hunk 1]
```diff
@@ -870,9 +874,7 @@ export class DumperState {
       } else {
         if (this.skipInvalid) return null;
         throw new TypeError(
-          `unacceptable kind of an object to dump ${
-            getObjectTypeString(value)
-          }`,
+          `Cannot dump object of type: ${getObjectTypeString(value)}`,
```
Comment (kt3k) [inline yaml/_dumper_state.ts:877]: The verb 'dump' is only used internally. The public API is named `stringify`. So let's use 'stringify' here.

```suggestion
          `Cannot stringify object of type: ${getObjectTypeString(value)}`,
```

File: yaml/_loader_state.ts [hunk 2]
```diff
@@ -1661,7 +1697,7 @@ function composeNode(
         }
       }
     } else {
-      return state.throwError(`unknown tag !<${state.tag}>`);
+      return state.throwError(`Unknown tag !<${state.tag}>`);
```
Comment (kt3k) [inline yaml/_loader_state.ts:1700]:
```suggestion
      return state.throwError(`Cannot resolve unknown tag !<${state.tag}>`);
```

**Comment by kt3k** [comment]
Some messages use `Error 'verb'-ing` form and some uses `Cannot 'verb'` form. I think these 2 forms are interchangeable. Should we align these 2 forms into the 1 form, or should we just allow using either of them?

**Comment by irbull** [comment]
> Some messages use `Error 'verb'-ing` form and some uses `Cannot 'verb'` form. I think these 2 forms are interchangeable. Should we align these 2 forms into the 1 form, or should we just allow using either of them?

I think consistency would be good here. I think we've used `Cannot 'verb'` more often, I'll take a pass and align them. We've come this far.

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**